src/Idris/Package/ToJson.idr

```diff
@@ -0,0 +1,116 @@
+module Idris.Package.ToJson
+
+import Idris.Package.Types
+import Libraries.Data.String.Extra
+import Data.List
+import Core.FC
+import Core.Name.Namespace
+import Data.Maybe
+
+%default total
+
+-- We don't bother with a robust JSON implementation
+-- for this one-off JSON serialization.
```

Comment:
famous last words 😅

Reply:
hah. I guess the irony is that I’m claiming JSON is so useful that this PR is worthwhile at the same time as I am commenting that JSON is going to be used so narrowly that it can be implemented ad-hoc.